app/src/main/res/values/strings.xml

```diff
@@ -646,6 +652,9 @@
     <string name="sub_content_filter">Subreddit content filter</string>
     <string name="submission_share_reddit_url">Share comments</string>
     <string name="filter_content">Filter this content</string>
+    <string name="content_to_hide">Content to hide</string>
+    <string name="in">in</string>
+    <string name="on">on</string>
```

Comment:
Could you use a getString(id, string) format rather than concatenating? Would be better for other languages